Install_or_Update.bat

```diff
@@ -1,8 +1,10 @@
-set PATH=%PATH%;%USERPROFILE%\anaconda3\Scripts;
-call activate.bat
-call conda remove --name geoapps --all
-call conda env update --file environment.yml  --prune
-call activate.bat geoapps
-call conda env config vars set KMP_WARNINGS=0
+set CONDA_EXE=conda
+:: if Conda executable is not in the PATH, uncomment and set the executable location below
+::set CONDA_EXE=%USERPROFILE%\AppData\Local\Continuum\anaconda3\Library\bin\conda.bat
+
+call %CONDA_EXE% remove --name geoapps --all --yes
+call %CONDA_EXE% env update --file environment.yml --prune
+call %CONDA_EXE% env config vars set KMP_WARNINGS=0 --name geoapps
+call %CONDA_EXE% activate geoapps && python -m pip install -e .
```

Comment:
Got an issue here with this line. The pip install of geoapps tries to re-install h5py from pip, which seems to crash on wheels: 
![image](https://user-images.githubusercontent.com/55204635/138142561-634420a8-e00f-499a-bcd6-8eef64e88061.png)

Reply:
Just need to change to this:
`pip install -e . --no-deps`

Reply:
strange, I did not need that.
pip normally detects the dependency is already there

Reply:
anyway, dependencies are installed already in the env, so `--no-deps` makes sense. Thanks !